### DISCUSSION TITLE
Welcome to OpenKYC Discussions!

### DISCUSSION THREAD

**Discussion by zhu-j-faceonlive** [FaceOnLive/ID-Verification-OpenKYC#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by denmgarcia** [comment]
hello, we are trying this out. however i dont know if I did wrong upon uploading the picture it says an error parsing id

**Comment by zhu-j-faceonlive** [comment]
Could you please try again?
If it still doesn't work, please contact us (contact@faceonlive.com) to resolve it.
Thanks.
